.ci/operatorhub.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+if [ -z ${COMMUNITY_OPERATORS_REPOSITORY} ]; then
+    COMMUNITY_OPERATORS_REPOSITORY="$(dirname $(dirname $(pwd)))/operator-framework/community-operators"
+    echo "COMMUNITY_OPERATORS_REPOSITORY not set, using ${COMMUNITY_OPERATORS_REPOSITORY}"
+fi
+
+if [ ! -d ${COMMUNITY_OPERATORS_REPOSITORY} ]; then
+    echo "${COMMUNITY_OPERATORS_REPOSITORY} doesn't exist, aborting."
+    exit 1
+fi
+
+OLD_PWD=$(pwd)
+VERSION=$(grep -v \# jaeger-operator.version)
+PKG_FILE=deploy/olm-catalog/jaeger-operator/jaeger-operator.package.yaml
+CSV_FILE=deploy/olm-catalog/jaeger-operator/${VERSION}/jaeger-operator.v${VERSION}.clusterserviceversion.yaml
+
+# once we get a clarification on the following item, we might not need to have different file names
+# https://github.com/operator-framework/community-operators/issues/701
+DEST_PKG_FILE=jaeger.package.yaml
+DEST_CSV_FILE=jaeger.v${VERSION}.clusterserviceversion.yaml
+
+cd "${COMMUNITY_OPERATORS_REPOSITORY}"
+
+git remote | grep upstream > /dev/null
+if [ $? != 0 ]; then
+    echo "Cannot find a remote named 'upstream'. Adding one."
+    git remote add upstream git@github.com:operator-framework/community-operators.git
+fi
+
+git fetch -q upstream
+git checkout -q master
+git rebase -q upstream/master
+
+for dest in upstream-community-operators community-operators; do
+    cp "${OLD_PWD}/${PKG_FILE}" "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${DEST_PKG_FILE}"
+    cp "${OLD_PWD}/${CSV_FILE}" "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${DEST_CSV_FILE}"
+
+    git checkout -q master
+
+    git checkout -q -b Update-Jaeger-${dest}-to-${VERSION}
+    if [ $? != 0 ]; then
+        echo "Cannot switch to the new branch Update-Jaeger-${dest}-to-${VERSION}. Aborting"
+        exit 1
+    fi
+
+    git add "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/"
+    git commit -qm "Update Jaeger ${dest} to v${VERSION}"
+    git push -q
+
+    command -v hub > /dev/null
+    if [ $? != 0 ]; then
+        echo "'hub' command not found, can't submit the PR on your behalf."
+        break
+    fi
+
+    tmpfile=$(mktemp /tmp/Update-Jaeger-${dest}-to-${VERSION}.XXX)
+    cat > ${tmpfile} <<- EOM
+Update Jaeger ${dest} to v${VERSION}
```

Comment:
Discussion point: How do we deal with the situation where we may overwrite their template? Might cause problems - but at the same time it would be problematic for us to try to auto respond to questions when we don't know how the template may change.

Thought: Can we have a mechanism that identifies when the template changes and abort until some steps are taken to align with the new template?

Reply:
Let me check, but if this template is stored as a file in their repo, we can keep a copy of it on our side, and do a diff before sending the PR.

Reply:
https://github.com/operator-framework/community-operators/blob/master/docs/pull_request_template.md